babel.config.js

```diff
@@ -1,6 +1,6 @@
-module.exports = function(api) {
+module.exports = function (api) {
   api.cache(true);
-  
+
```

Comment:
Since I committed this file after reverting my changes, our pre-commit hooks wanted to run Prettier on it